src/main/server.ts

```diff
@@ -18,12 +25,36 @@ if (app.locals.ENV === 'development') {
     cert: fs.readFileSync(path.join(sslDirectory, 'localhost.crt')),
     key: fs.readFileSync(path.join(sslDirectory, 'localhost.key')),
   };
-  const server = https.createServer(sslOptions, app);
-  server.listen(port, () => {
+  httpsServer = https.createServer(sslOptions, app);
+  httpsServer.listen(port, () => {
     logger.info(`Application started: https://localhost:${port}`);
   });
 } else {
-  app.listen(port, () => {
+  httpServer = app.listen(port, () => {
```

Comment:
This looks like a regression, the previous code only created a HTTPS server and never created a HTTP server. Currently this code will only create a HTTPS server if it is running on a development environment.